src/Voxel.MiddyNet/MiddyNet.cs

```diff
@@ -68,8 +82,8 @@ private void InitialiseMiddyContext(ILambdaContext context)
             {
                 MiddyContext.AttachToLambdaContext(context);
             }
-
-            MiddyContext.AdditionalContext.Clear(); //  Given that the instance is reused, we need to clean the dictionary.
+            
+            MiddyContext.Clear();
```

Comment:
Can we keep the comment? I think it might be important here.

Reply:
Sure.

Reply:
I moved it into the MiddyContext.Clear() method.

Reply:
To be picky, I think it makes more sense to keep it here, not there :P

Reply:
Ok, I'll make the change.

Reply:
See if it suits you now, please.